cluster-autoscaler/cloudprovider/huaweicloud/huaweicloud_service_manager.go

```diff
@@ -626,10 +627,43 @@ func (csm *cloudServiceManager) buildNodeFromTemplate(asgName string, template *
 
 	node.Labels = cloudprovider.JoinStringMaps(node.Labels, buildGenericLabels(template, nodeName))
 
+	node.Spec.Taints = extractTaintsFromTags(template.tags)
+
 	node.Status.Conditions = cloudprovider.BuildReadyConditions()
 	return &node, nil
 }
 
+func extractTaintsFromTags(tags map[string]string) []apiv1.Taint {
```

Comment:
Can you give an example of the tag's format? 
It's worth adding unit tests for this.

Reply:
done